Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -56,6 +57,17 @@
 
 #define MRC_PARAMS_BYTE_OFFSET_MRC_VERSION 14
 
+SBL_DEVICE  mSblDevices[]= {
+  // Device type     instance  rsvd   fun  dev  bus  ismmio
+  {OsBootDeviceSata  , 0,       0,   {{ 0,  18,   0,    0 }} },
```

Comment:
why don't you use a macro for each device info?
ex) PCI_DEVICE(b, d, f), ADDR_DEVICE(addr)

Reply:
Will refine the structure and use a new method to avoid a new macro.